app/base/static/assets/css/lightbox.css

```diff
@@ -0,0 +1,204 @@
+body.lb-disable-scrolling {
```

Comment:
move all the css and js assets to base folder of `app/base/static/css/` and `app/base/static/js/` respectively

Reply:
I have moved them.